### PR TITLE
#11929: Move work_split_tilize

### LINKED ISSUE #11929: Move `work_split_tilize`
Tracking issue : https://github.com/tenstorrent/tt-metal/issues/11688 , Master Issue : https://github.com/tenstorrent/tt-metal/issues/11513

Status : PR Review

### PR DESCRIPTION
**Pull request by VirdhatchaniKN**
Issue : #11929 , Tracking issue : https://github.com/tenstorrent/tt-metal/issues/11688 , Master Issue : https://github.com/tenstorrent/tt-metal/issues/11513

CI : 

- [All post-commit tests](https://github.com/tenstorrent/tt-metal/actions/runs/10574769478) - PASSED
-  [[post-commit] C++ tests](https://github.com/tenstorrent/tt-metal/actions/runs/10575779727) - PASSED
- [(Single-card) Device perf regressions](https://github.com/tenstorrent/tt-metal/actions/runs/10575780635)
-  [[post-commit] models tests](https://github.com/tenstorrent/tt-metal/actions/runs/10575781979) - PASSED
-  [(Single-card) Model perf tests](https://github.com/tenstorrent/tt-metal/actions/runs/10582583963) - PASSED
-  [(T3K) T3000 model perf tests](https://github.com/tenstorrent/tt-metal/actions/runs/10575786242) - PASSED
-  [(T3K) T3000 unit tests](https://github.com/tenstorrent/tt-metal/actions/runs/10575787272) - PASSED
-  [(T3K) T3000 demo tests](https://github.com/tenstorrent/tt-metal/actions/runs/10575789936)
-  [Nightly fast dispatch tests](https://github.com/tenstorrent/tt-metal/actions/runs/10575791300)
-  [(T3K) T3000 frequent tests](https://github.com/tenstorrent/tt-metal/actions/runs/10575792691)
-  [(Single-card) Demo tests](https://github.com/tenstorrent/tt-metal/actions/runs/10575793770) - PASSED

### CODE REVIEW

**Review by ntarafdar — APPROVED** [review]

**Review by cfjchu — APPROVED** [review]

**Review by patrickroberts — APPROVED** [review]

**Review by sjameelTT — APPROVED** [review]

File: ttnn/cpp/ttnn/operations/core/work_split/work_split_tilize.hpp [hunk 1]
```diff
@@ -228,4 +230,8 @@ inline std::vector<std::vector<BlockRep>> distribute_work(
     return core_assignments;
 }
 
-}  // namespace tt::tt_metal
+}
```
Comment (KalaivaniMCW) [inline ttnn/cpp/ttnn/operations/core/work_split/work_split_tilize.hpp:233]: // namespace operations::core::work_split

**Review by KalaivaniMCW — APPROVED** [review]
Looks good. left one comment for minor change. Approved.

File: ttnn/cpp/ttnn/operations/core/work_split/work_split_tilize.hpp (hunk 1 above)
Comment (VirdhatchaniKN) [reply ttnn/cpp/ttnn/operations/core/work_split/work_split_tilize.hpp]: Addressed

**Review by dmakoviichuk-tt — APPROVED** [review]

**Comment by VirdhatchaniKN** [comment]
https://github.com/tenstorrent/tt-metal/actions/runs/10589536095 - passed

**Pull request merged**